src/utils/getCollection.js

```diff
@@ -0,0 +1,7 @@
+import { getCollection as get } from 'astro:content';
+
+export async function getCollection(collection) {
+  return await get(collection, ({ data }) => {
+    return import.meta.env.MODE === 'production' ? data.draft !== true : true;
```

Comment:
@AndyClifford This can be simplified

Change: `return import.meta.env.MODE !== 'production' || !data.draft`